internal/aiptest/search/search.go

```diff
@@ -27,22 +25,13 @@ var Suite = suite.Suite{
 var withResourcesGroup = suite.TestGroup{
 	OnlyIf: suite.OnlyIfs(
 		onlyif.HasParent,
-		onlyif.HasMethod(aipreflect.MethodTypeCreate),
-		onlyif.MethodNotLRO(aipreflect.MethodTypeCreate),
```

Comment:
Was the removal of the LRO guard here intentional?

Reply:
Yes, the test will be skipped here
https://github.com/einride/protoc-gen-go-aip-test/pull/86/files#diff-72594cd72ecd693b605c65c096c34fd6d3c78371c42cc2e8dc5206a034eec738R220

Could still keep the LRO guard here and not generate the method, but I found it confusing to keep the LRO check if the method check is removed.